### ISSUE TITLE
Request: Add integration with Stripe Elements

### ISSUE THREAD

**Issue by vmitchell85** [muhammederdem/credit-card-form#7]
It would be really awesome if we could integrate this UI with [Stripe Elements](https://stripe.com/payments/elements).

I'm adding this issue in case someone else picks it up before I can attempt it.

**Comment by damcclean** [comment]
Miguel Piedraftia actually managed to do this yesterday and he made a write up on it: https://miguelpiedrafita.com/stripe-elements-card/

**Comment by sblfc** [comment]
> It would be really awesome if we could integrate this UI with [Stripe Elements](https://stripe.com/payments/elements).
> 
> I'm adding this issue in case someone else picks it up before I can attempt it.

I too would find this useful.

**Comment by vmitchell85** [comment]
There was a comment on the post by @m1guelpf asking about his source code, but no answer yet. I'll comment here if I get it working

**Comment by m1guelpf** [comment]
@vmitchell85 As I mentioned on the comments I'll try to decouple the component from the [Sitesauce](https://sitesauce.app) this week and throw it in a repo, but don't have time today unfortunately :)

**Comment by vmitchell85** [comment]
@m1guelpf sweet, it didn’t notify me of your comment since it was in a sub-comment... I might give it a shot anyway... thanks

**Comment by sblfc** [comment]
Any updates on this?

**Comment by vmitchell85** [comment]
Sorry @sblfc I haven't gotten around to it...

**Comment by Piket564** [comment]
@vmitchell85 @sblfc I've managed to insert Stripe with /api/v2

**Comment by dominik-wbz** [comment]
I don't think it's actually possible to do with Stripe.js v3. What is possible is what m1guelpf had done and that's to place the inputs inside the card. However that's far from optimal imho :(

**Comment by Piket564** [comment]
> I don't think it's actually possible to do with Stripe.js v3. What is possible is what m1guelpf had done and that's to place the inputs inside the card. However that's far from optimal imho :(

Yes, Stripe provide hi own input for security concern, they do not limit the CSS though, i've tried what m1guelpf had done, and yes is a little tricky but it works...

**Comment by callsecfrance** [comment]
Hi everyone,

Any update?

**Comment by AxelTheGerman** [comment]
I gave up on this, it's a nice visual but not worth the effort to me. It's really neat but I'm gonna ship more features instead :)
